README.md

```diff
@@ -23,31 +26,29 @@ $ npm install @gh-conf/gh-conf-validate
 ## Usage
 
 ```javascript
-const { validatePath } = require('@gh-conf/gh-conf-validate');
+const { validatePath } = require("@gh-conf/gh-conf-validate");
 
 // Valid path
-const isValid = validatePath('./gh-conf-validate');
+const isValid = validatePath("./gh-conf-validate");
 console.log(isValid);
 // Output
 // true
 
-
 // Invalid path
 try {
-  validatePath()
+	validatePath();
```

Comment:
We use 2 spaces not tag spaces. Semicolon is fine though.

Same for below lines also

Reply:
Again my editor auto-formatting, will look asap!